Add: Footer Content

Nickyshe opened the pull request:
What Was Migrated
* About us page
* Build your portfolio page
* Trusted talent page

Testing 
* Links work as expected
* Pages are rendering properly

Nickyshe commented:
@Aahil13 Kindly review my pull request

Aahil13 commented:
<img width="1304" alt="Screenshot 2025-05-11 at 9 46 48 PM" src="https://github.com/user-attachments/assets/345416a4-0e39-4064-b325-373f5dde88b5" />
<img width="1303" alt="Screenshot 2025-05-11 at 9 46 59 PM" src="https://github.com/user-attachments/assets/7a7cc384-78b6-4e18-98ac-c7f9fd46e51f" />

If you notice from these screenshots, the tabs feature on the trusted talents page doesn't show the text in light mode, but it does in dark mode. So let's fix that.

Also, not an issue, but if it's something that can be done quickly, i.e in under 5 mins, instead of using a list with links for the build your portfolio page, you can use cards for API and Glasp. If not, send the PR without it, and I'll merge.

Nickyshe commented:
> <img alt="Screenshot 2025-05-11 at 9 46 48 PM" width="1304" src="https://private-user-images.githubusercontent.com/63567230/442535125-345416a4-0e39-4064-b325-373f5dde88b5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDcwMjczNDYsIm5iZiI6MTc0NzAyNzA0NiwicGF0aCI6Ii82MzU2NzIzMC80NDI1MzUxMjUtMzQ1NDE2YTQtMGUzOS00MDY0LWIzMjUtMzczZjVkZGU4OGI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTEyVDA1MTcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViYmEyMzViMjc1OTUwMGM4OGZhODhlZTE5ZmJmNmY0YmZkYzcwMzNiMWUyNTRiYWMwNjQyYWMzOTE4OTkzNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Y_ehlT2h0ggNQPRFgcmhtRAUGEak5I_-LRk3eJx49dY"> <img alt="Screenshot 2025-05-11 at 9 46 59 PM" width="1303" src="https://private-user-images.githubusercontent.com/63567230/442535128-7a7cc384-78b6-4e18-98ac-c7f9fd46e51f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDcwMjczNDYsIm5iZiI6MTc0NzAyNzA0NiwicGF0aCI6Ii82MzU2NzIzMC80NDI1MzUxMjgtN2E3Y2MzODQtNzhiNi00ZTE4LTk4YWMtYzdmOWZkNDZlNTFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTEyVDA1MTcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4OWY0NWVhYWQ5YjRlYzNlMWI4ZjYzNzQ2ZDNhNmE3ODVmN2FhZThiZWQyNTEwODFiOGRlNmIzNmMwODE3ZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SFGVliGetPgpJ-vXzNrBRvzhsM435SWL91-4rXVGfJk">
> If you notice from these screenshots, the tabs feature on the trusted talents page doesn't show the text in light mode, but it does in dark mode. So let's fix that.
> 
> Also, not an issue, but if it's something that can be done quickly, i.e in under 5 mins, instead of using a list with links for the build your portfolio page, you can use cards for API and Glasp. If not, send the PR without it, and I'll merge.

I have fixed the light and dark mode issue with the trusted talent page. I also changed the list to cards in the build your portfolio page. Kindly review again @Aahil13

Aahil13 commented:
LGTM!!!
